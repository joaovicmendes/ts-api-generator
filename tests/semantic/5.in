// Deve reportar que o identificador Ã© uma classe do modelo
model [
    Student {
        name: string
        email: string
        ra: number
    }
    ExampleClass {
        student: Student
        grade: number
    }
]
endpoints [
    Student {
        GET '/students'
        GET '/students/:ra'
        POST '/students'
        PUT '/students/:ra'
        DELETE '/students/:ra'
    }
    ExampleClass {
        GET 'class/:student'
    }
]
