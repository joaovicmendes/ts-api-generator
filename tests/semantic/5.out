Linha 22: tipos declarados no modelo não são permitidos como identificador de rotas
Compilação interrompida.
